app/components/DocumentCopy.tsx

```diff
@@ -46,19 +46,13 @@ function DocumentCopy({ document, onSubmit }: Props) {
     return nodes;
   }, [policies, collectionTrees, document.isTemplate]);
 
-  const handlePublishChange = React.useCallback(
-    (ev: React.ChangeEvent<HTMLInputElement>) => {
-      setPublish(ev.target.checked);
-    },
-    []
-  );
+  const handlePublishChange = React.useCallback((checked: boolean) => {
```

Comment:
Remove all callbacks that are no longer needed – as the boolean can be passed straight to the `setPublish` method.